engine/src/test/java/org/hibernate/validator/test/internal/engine/ValidatorTest.java

```diff
@@ -325,6 +344,20 @@ public String getFoo() {
 		}
 	}
 
+	class I {
+		@Valid
+		Map<String, J> map = new HashMap<String, J>();
```

Comment:
HI, it's more of a question :) - I've noticed that there's this `CollectionHelper` util class and sometimes in the code it is used to create different collections with for example `CollectionHelper#newHashMap` and some times jus simple instantiation is used like here. Is there any preferences/recommendations when to use what, or is it more of style thing ?